samples/highcharts/chartchooser/categorical-composition-stacked-column-simple/demo.js

```diff
@@ -0,0 +1,93 @@
+Highcharts.chart("container", {
+    chart: {
+        type: "column"
+    },
+    title: {
+        text: "Earth's atmosphere"
+    },
+    subtitle: {
+        text:
+      'Source: <a href="https://www.nasa.gov/mission_pages/sunearth/science/atmosphere-layers2.html">nasa.gov</a>'
+    },
+    xAxis: {
+        categories: ["Earth's atmosphere"],
+        visible: false
+    },
+    yAxis: {
+        labels: {
+            enabled: false
+        },
+        visible: false,
+        min: 0,
+        title: {
+            text: null
+        }
+    },
+    tooltip: {
+        headerFormat: null,
+        pointFormat: "{point.series.userOptions.info}"
```

Comment:
This works, but:
* Use `options` instead of `userOptions`
* In respect of the TypeScript folks, use the [custom](https://api.highcharts.com/highcharts/series.line.custom) option.

Reply:
@mekhatria If you fix this then we can finalize the PR